handlers/messagebird/messagebird_test.go

```diff
@@ -0,0 +1,231 @@
+		MsgText:             "Simple Message ☺",
+		MsgURN:              "tel:188885551515",
+		MsgAttachments:      []string{"image/jpeg:https://foo.bar/image.jpg"},
+		MockResponseBody:    "",
+		MockResponseStatus:  200,
+		ExpectedHeaders:     map[string]string{"Content-Type": "application/json", "Authorization": "AccessKey authtoken"},
+		ExpectedRequestBody: `{"recipients":["188885551515"],"originator":"18005551212","body":"Simple Message ☺","mediaUrls":["https://foo.bar/image.jpg"]}`,
+		ExpectedMsgStatus:   "W",
+		ExpectedExternalID:  "",
+		SendPrep:            setMmsSendURL,
+	},
+	{
+		Label:               "Send with image only",
+		MsgURN:              "tel:188885551515",
+		MsgAttachments:      []string{"image/jpeg:https://foo.bar/image.jpg"},
+		MockResponseBody:    "",
+		MockResponseStatus:  200,
+		ExpectedHeaders:     map[string]string{"Content-Type": "application/json", "Authorization": "AccessKey authtoken"},
+		ExpectedRequestBody: `{"recipients":["188885551515"],"originator":"18005551212","mediaUrls":["https://foo.bar/image.jpg"]}`,
+		ExpectedMsgStatus:   "W",
+		ExpectedExternalID:  "",
+		SendPrep:            setMmsSendURL,
+	},
+	{
+		Label:               "Send with two images",
+		MsgURN:              "tel:188885551515",
+		MsgAttachments:      []string{"image/jpeg:https://foo.bar/image.jpg", "image/jpeg:https://foo.bar/image2.jpg"},
+		MockResponseBody:    "",
+		MockResponseStatus:  200,
+		ExpectedHeaders:     map[string]string{"Content-Type": "application/json", "Authorization": "AccessKey authtoken"},
+		ExpectedRequestBody: `{"recipients":["188885551515"],"originator":"18005551212","mediaUrls":["https://foo.bar/image.jpg","https://foo.bar/image2.jpg"]}`,
+		ExpectedMsgStatus:   "W",
+		ExpectedExternalID:  "",
+		SendPrep:            setMmsSendURL,
+	},
+	{
+		Label:               "Send with video only",
+		MsgURN:              "tel:188885551515",
+		MsgAttachments:      []string{"video/mp4:https://foo.bar/movie.mp4"},
+		MockResponseBody:    "",
+		MockResponseStatus:  200,
+		ExpectedHeaders:     map[string]string{"Content-Type": "application/json", "Authorization": "AccessKey authtoken"},
+		ExpectedRequestBody: `{"recipients":["188885551515"],"originator":"18005551212","mediaUrls":["https://foo.bar/movie.mp4"]}`,
+		ExpectedMsgStatus:   "W",
+		ExpectedExternalID:  "",
+		SendPrep:            setMmsSendURL,
+	},
+	{
+		Label:               "Send with pdf",
+		MsgURN:              "tel:188885551515",
+		MsgAttachments:      []string{"application/pdf:https://foo.bar/document.pdf"},
+		MockResponseBody:    "",
+		MockResponseStatus:  200,
+		ExpectedHeaders:     map[string]string{"Content-Type": "application/json", "Authorization": "AccessKey authtoken"},
+		ExpectedRequestBody: `{"recipients":["188885551515"],"originator":"18005551212","mediaUrls":["https://foo.bar/document.pdf"]}`,
+		ExpectedMsgStatus:   "W",
+		ExpectedExternalID:  "",
+		SendPrep:            setMmsSendURL,
+	},
```

Comment:
We need to tests for errors too, simulate like 400, 500 status response and make sure the message status is correctly saved

Reply:
added